app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -116,49 +156,776 @@ class AdministratorControlsFragmentTest {
+    launch<AdministratorControlsActivity>(
+      createAdministratorControlsActivityIntent(
+        profileId = internalProfileId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      verifyItemDisplayedOnAdministratorControlListItem(
+        itemPosition = 0,
+        targetView = R.id.general_text_view
+      )
+      verifyTextOnAdministratorListItemAtPosition(
+        itemPosition = 0,
+        targetViewId = R.id.edit_account_text_view,
+        stringIdToMatch = R.string.administrator_controls_edit_account
+      )
+      verifyItemDisplayedOnAdministratorControlListItem(
+        itemPosition = 1,
+        targetView = R.id.profile_management_text_view
+      )
+      verifyTextOnAdministratorListItemAtPosition(
+        itemPosition = 1,
+        targetViewId = R.id.edit_profiles_text_view,
+        stringIdToMatch = R.string.administrator_controls_edit_profiles
+      )
+    }
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_downloadPermissionsAndSettingsIsDisplayed() {
+    launch<AdministratorControlsActivity>(
+      createAdministratorControlsActivityIntent(
+        profileId = internalProfileId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      verifyTextOnAdministratorListItemAtPosition(
+        itemPosition = 2,
+        targetViewId = R.id.download_permissions_text_view,
+        stringIdToMatch = R.string.administrator_controls_download_permissions_label
+      )
+      verifyItemDisplayedOnAdministratorControlListItem(
+        itemPosition = 2,
+        targetView = R.id.topic_update_on_wifi_constraint_layout
+      )
+      scrollToPosition(position = 2)
+      verifyItemDisplayedOnAdministratorControlListItem(
+        itemPosition = 2,
+        targetView = R.id.auto_update_topic_constraint_layout
+      )
+    }
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_applicationSettingsIsDisplayed() {
+    launch<AdministratorControlsActivity>(
+      createAdministratorControlsActivityIntent(
+        profileId = internalProfileId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      scrollToPosition(position = 3)
+      verifyItemDisplayedOnAdministratorControlListItem(
+        itemPosition = 3,
+        targetView = R.id.app_information_text_view
+      )
+      verifyTextOnAdministratorListItemAtPosition(
+        itemPosition = 3,
+        targetViewId = R.id.app_version_text_view,
+        stringIdToMatch = R.string.administrator_controls_app_version
+      )
+      verifyItemDisplayedOnAdministratorControlListItem(
+        itemPosition = 4,
+        targetView = R.id.account_actions_text_view
+      )
+      verifyTextOnAdministratorListItemAtPosition(
+        itemPosition = 4,
+        targetViewId = R.id.log_out_text_view,
+        stringIdToMatch = R.string.administrator_controls_log_out
+      )
+    }
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_wifiSwitchIsUncheck_autoUpdateSwitchIsUncheck() {
```

Comment:
```suggestion
  fun testAdministratorControlsFragment_wifiSwitchIsUnchecked_autoUpdateSwitchIsUnchecked() {
```

Ditto elsewhere (where relevant).

Reply:
done